docs/api_docs/python/compiler/build.rst

```diff
@@ -0,0 +1,9 @@
+Build API
+=======================
+
+.. automodule:: iree.build
+  :members:
+  :imported-members:
+  :undoc-members:
+
+.. TODO: organize into sections (core executor, arguments, built in actions, etc.)
```

Comment:
Preview as this is now: 
![image](https://github.com/user-attachments/assets/cb7788e0-8bf8-4ec8-ba39-725ed24941b7)